assets/training/finetune_acft_image/environments/acpt_image_huggingface/asset.yaml

```diff
@@ -0,0 +1,6 @@
+name: acft-transformers-image-gpu
```

Comment:
should this be acpt?

Reply:
would be good to have some description for what is acpt or acft :)

Reply:
acpt - Azure Container for PyTorch
acft - azure container for finetuning